wallets/rn_cli_wallet/src/screens/SubscriptionDetailsScreen.tsx

```diff
@@ -0,0 +1,146 @@
+import * as React from 'react';
+import {useRoute} from '@react-navigation/native';
+import {Alert, FlatList, View} from 'react-native';
+import useNotifyClientContext from '@/hooks/useNotifyClientContext';
+
+import NotificationItem from '@/components/NotificationItem';
+import {useTheme} from '@/hooks/useTheme';
+
+interface NotifyNotification {
+  title: string;
+  sentAt: number;
+  body: string;
+  id: string;
+  url: string | null;
+  type: string;
+}
+
+function NotificationItemSkeleton() {
```

Comment:
I noticed you have two skeleton components, maybe you could create a generic one and reuse it when needed. You can check this one that uses react-native-svg to create the loading effect: https://github.com/WalletConnect/web3modal-react-native/blob/1ce16148e5daa75950255b6a8e02b65e491ebec0/packages/ui/src/components/wui-shimmer/index.tsx



and its used [here](https://github.com/WalletConnect/web3modal-react-native/blob/1ce16148e5daa75950255b6a8e02b65e491ebec0/packages/ui/src/composites/wui-card-select-loader/index.tsx)

Reply:
Good call, I think we don't need second one